### ISSUE TITLE
Crash when exporting layout to picture

### ISSUE THREAD

**Issue by GitHubMLee** [qgis/QGIS#52136]
### What is the bug or the crash?

QGIS crashes unexpectedly when trying to export layout as png file.

### Steps to reproduce the issue

I have a relatively simple map with road layers, a zone layer (at 50% transparency using a thematic representation) and a subject site outlet. All of this is displayed over a Google Satellite image. The map is shown at EPSG 3857 (as per Google).
The subject site is labelled with a bubble callout style, with the pointer set to the exterior of the shape.
The zone map has zone code labels with a minimum size set to exclude very small zone regions.
Have created the layout window and am trying to export at 300 dpi and with antialias.
I click the 'Export to picture' button. Choose 300 dpi and antialias (this is my default anyway and usually doesn't cause a problem) and click save and then there is a crash after about 5-10 secs.
Crashes in both 3.28.4 (LTR) and 3.30.

### Versions

3.28.4

### Supported QGIS version

- [X] I'm running a supported QGIS version according to the roadmap.

### New profile

- [ ] I tried with a new QGIS profile

### Additional context

Copy report output

## User Feedback

Trying to export layout as png picture.

## Report Details

**Python Stack Trace**
```
Windows fatal exception: access violation

Current thread 0x00002b6c (most recent call first):
<no Python frame>
```

**Stack Trace**

```

QPainterPath::addPath :
QgsSymbolLayer::prepareMasks :
QgsSymbol::startRender :
QgsSingleSymbolRenderer::startRender :
QgsVectorLayerProfileResults::visitFeaturesInRange :
QgsVectorLayerProfileResults::visitFeaturesInRange :
QgsMapRendererCustomPainterJob::doRender :
QgsMapRendererCustomPainterJob::startPrivate :
QgsMapRendererCustomPainterJob::renderSynchronously :
QgsLayoutItemMap::drawMap :
QgsLayoutItemMap::paint :
QGraphicsScene::dragMoveEvent :
QGraphicsItem::dragMoveEvent :
QGraphicsEffectSource::draw :
QgsLayoutEffect::draw :
QGraphicsScene::drawItems :
QGraphicsScene::drawItems :
QGraphicsScene::render :
QgsLayoutExporter::renderRegion :
QgsLayoutExporter::renderRegionToImage :
QgsLayoutExporter::renderPageToImage :
QgsLayoutExporter::createImage :
QgsLayoutExporter::exportToImage :
QgsGeoreferencerMainWindow::dragEnterEvent :
QObject::qt_static_metacall :
QAction::activate :
QAbstractButton::click :
QAbstractButton::mouseReleaseEvent :
QToolButton::mouseReleaseEvent :
QWidget::event :
QApplicationPrivate::notify_helper :
QApplication::notify :
QgsApplication::notify :
QCoreApplication::notifyInternal2 :
QApplicationPrivate::sendMouseEvent :
QSizePolicy::QSizePolicy :
QSizePolicy::QSizePolicy :
QApplicationPrivate::notify_helper :
QApplication::notify :
QgsApplication::notify :
QCoreApplication::notifyInternal2 :
QGuiApplicationPrivate::processMouseEvent :
QWindowSystemInterface::sendWindowSystemEvents :
QEventDispatcherWin32::processEvents :
qt_plugin_query_metadata :
QEventLoop::exec :
QCoreApplication::exec :
main :
BaseThreadInitThunk :
RtlUserThreadStart :
```

**QGIS Info**
QGIS Version: 3.28.4-Firenze
QGIS code revision: fd0fb72ab3
Compiled against Qt: 5.15.3
Running against Qt: 5.15.3
Compiled against GDAL: 3.6.2
Running against GDAL: 3.6.2

**System Info**
CPU Type: x86_64
Kernel Type: winnt
Kernel Version: 10.0.22621

**Comment by roya0045** [comment]
In the layout, if you select export as raster does that change anything? Is the issue only with png format?

**Comment by GitHubMLee** [comment]
I tested the issue with a new User profile and was able to export to png successfully. However, this profile had an issue with semi-transparent polygon shading overlaid on top of a raster background - the polygons were filled in (100% opaque). However, by selecting export as raster the maps then exported successfully.
I also went back to my original profile and disabled all plugins other than key/core plugins, and I was able to export to png. I haven't yet gone through the process of progressively turning on individual plugins to see which was triggering the problem...
